samples/Linq/SvgExample/StronglyTypedTools.cs

```diff
@@ -20,15 +20,20 @@ public static void AddSvg(Stream stream, string svgPath, double percentageOfCy)
         {
             using PresentationDocument presentationDocument = PresentationDocument.Open(stream, true);
             PresentationPart presentationPart = presentationDocument.PresentationPart ??
-                                                throw new InvalidOperationException(@"PresentationDocument is invalid.");
+                                                throw new InvalidOperationException("PresentationDocument is invalid.");
+
+            if (presentationPart.Presentation is null)
+            {
+                throw new ArgumentNullException("Presentation root element is missing!");
+            }
 
             // Get relationship ID of first slide.
             string sldRelId = presentationPart
                 .Presentation
                 .SlideIdList?
                 .Elements<Presentation.SlideId>()
                 .Select(slideId => (string)slideId.RelationshipId!)
-                .FirstOrDefault() ?? throw new InvalidOperationException(@"Presentation has no slides.");
+                .FirstOrDefault() ?? throw new InvalidOperationException("Presentation has no slides.");
```

Comment:
if you want to ensure it's there, you can just call `.First()` and it will throw if it's not available